include/lsst/jointcal/Associations.h

```diff
@@ -81,12 +81,17 @@ class Associations {
     /**
      * @brief      Collect stars from an external reference catalog and associate them with fittedStars.
      *
-     * @param      refCat     The catalog of reference sources
-     * @param[in]  matchCut   Separation radius to match fitted and reference stars.
-     * @param      fluxField  The field name in refCat to get the flux from.
+     * @param      refCat         The catalog of reference sources
+     * @param[in]  matchCut       Separation radius to match fitted and
+     *                            reference stars.
+     * @param      fluxField      The field name in refCat to get the flux from.
+     * @param      refFluxMap     fluxes per filter of corresponding refCat objects (can be empty)
+     * @param      refFluxErrMap  flux errors per filter of corresponding refCat objects (can be empty)
      */
     void collectRefStars(lsst::afw::table::SortedCatalogT<lsst::afw::table::SimpleRecord> &refCat,
-                         afw::geom::Angle matchCut, std::string const &fluxField);
+                         afw::geom::Angle matchCut, std::string const &fluxField,
+                         std::map<std::string, std::vector<double> > const &refFluxMap,
+                         std::map<std::string, std::vector<double> > const &refFluxErrMap);
```

Comment:
I find it helps readability for me, otherwise I have trouble separating the brackets. We don't seem to have a documented standard on this.